### DISCUSSION TITLE
My system cannot find the colmap command

### DISCUSSION THREAD

**Discussion by SsNiPeR1** [NVlabs/instant-ngp#477] [Q&A]
My system cannot find the colmap command, when i run colmap2nerf.py

`running: colmap feature_extractor --ImageReader.camera_model OPENCV --SiftExtraction.estimate_affine_shape=true --SiftExtraction.domain_size_pooling=true --ImageReader.single_camera 1 --database_path colmap.db --image_path images`
how to install that command, or where is the binary for colmap?

**Comment by YuhsiHu** [comment]
`
sudo apt install colmap
`

**Comment by SsNiPeR1** [comment]
> `sudo apt install colmap`

i hate Debian-based systems, and i use gentoo, alr found a way, but it does not work.

**Reply by ashishd** [reply]
Refer to 
https://colmap.github.io/install.html
for binary or source installations
